modules/AdvancedPracticeMode/Gui.cpp

```diff
@@ -324,13 +327,25 @@ namespace Practice
 		SokuLib::activeWeather = SokuLib::WEATHER_CLEAR;
 		SokuLib::weatherCounter = 999;
 	}
-
+	
 	static void displayStatePanel(const std::string &profile)
 	{
 		panel->loadWidgetsFromFile(profile + "/assets/state.gui");
 
 		auto force = panel->get<tgui::CheckBox>("ForceWeather");
 		auto weather = panel->get<tgui::ComboBox>("Weather");
+		auto editTimer = panel->get<tgui::EditBox>("EditTimer");
+
+
+			editTimer->connect("ReturnKeyPressed", [](std::string time) {
+				int intTime = 999;
+			
+				if (time.length() != 0) {
+					intTime = stoi(time);
+					settings.weatherTime = intTime;
```

Comment:
I think it would be better to have the actual number on the timer (with the dot) to be inputed in the box because writting 89 to have the timer at 8.9 is a bit weird